tests/unittests/tests-hashes/tests-hashes.h

```diff
@@ -79,7 +79,7 @@ Test *tests_hashes_sha256_hmac_tests(void);
  */
 Test *tests_hashes_sha256_chain_tests(void);
 
-  /**
+/**
```

Comment:
This change is unrelated. We can keep it in this PR but it should be in its own commit.

Reply:
Alright.